src/libraries/Microsoft.Extensions.Logging/tests/Common/ConsoleLoggerTest.cs

```diff
@@ -1032,7 +1032,7 @@ public void ConsoleLoggerOptions_DisableColors_IsAppliedToLoggers()
         }
 
         [Fact]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/38337", TestPlatforms.Browser)]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
```

Comment:
you can't have both `Fact` and `ConditionalFact`, only the latter one is enough.

I also see a few more tests in this file that are disabled against #38337, could you please check whether they're "fixed" as well?